src/bin/ion/commands/beta/generate/templates/java/class.templ

```diff
@@ -16,4 +21,110 @@ public final class {{ target_kind_name }} {
         return this.{{ field.name | camel }};
     }
     {% endfor %}
+
+    public static {{ target_kind_name }} readFrom(IonReader reader) {
+        {# Intializes all the fields of this class #}
+        {% for field in fields -%}
+        {{ field.value }} {{ field.name | camel }} =
```

Comment:
Is `field.value` the _type_ of the field? (If so, could it be named `value_type` instead?)

Reply:
Yes, it is the type of field. I will rename it to `value_type`.